src/database/models.js

```diff
@@ -0,0 +1,252 @@
+const { DataTypes, Sequelize } = require('sequelize');
+
+/**
+ * SSL is required for Heroku Postgres
+ */
+const sslOptions = process.env.NODE_ENV === 'production' ?
+    { dialectOptions: { ssl: { rejectUnauthorized: false }, }, } :
+    {};
+
+const sequelize = new Sequelize(
+    process.env.DATABASE_URL,
+    {
+        logging:  false,
+        protocol: 'postgres',
+        dialect:  'postgres',
+        ...sslOptions,
+        pool:     {
+            max:     5,
+            min:     0,
+            acquire: 30000,
+            idle:    10000
+        }
+    }
+);
+
```

Comment:
Would suggest injecting the sequelize object into this file; basically, turn the whole file into one big function that accepts the `sequelize` instance object as a parameter, and internally calls `sequelize.define` for all the models.

This is kind of what I mean, generally: https://github.com/sequelize/express-example/blob/7304dc6ff4b842b06301d962bd0bcb07454042b8/express-main-example/sequelize/index.js

So then we'd have a separate file that did the work above and exported the sequelize instance. Somewhere near the start of the server, we import that file to get the sequelize instance, and pass it into this new `apply and make the models` function. Probably call `sequelize.sync()` then. At that point, we have a handle on the sequelize instance near the top of the program (and can pass it down to where we need it), and can access the models on it with `sequelize.models.Whatever`.